test_conformance/extensions/cl_khr_kernel_clock/kernel_clock.cpp

```diff
@@ -109,14 +109,10 @@ class KernelClockTest {
 
             ptr = kernel_src;
 
-            error = create_single_kernel_helper_create_program(
-                context, &program, 1, &ptr);
+            error = create_single_kernel_helper(context, &program, &kernel, 1,
+                                                &ptr, "SampleClock");
```

Comment:
I think we can remove the explicit call to `clCreateKernel` on Line 120 below now, also?

Reply:
Good catch, thanks!  Updated.